azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst

```diff
@@ -0,0 +1,137 @@
+Provision an Ubuntu virtual machine running SQL Server in Azure
+===============================================================
+
+This documentation is based on the `Ubuntu tutorial <https://ubuntu.com/tutorials/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure#2-launch-an-ubuntu-vm-with-sql-server-installed>`_
+for creating Linux virtual machines with the Azure CLI.
+
+
+Prerequisites
+-------------
+
+- A Microsoft Azure account
+
+
+Launch an Ubuntu VM with SQL Server installed
+---------------------------------------------
+
+
+Find the correct image to launch on the Azure portal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Sign in to the Azure portal. https://portal.azure.com/
+
+- In the left pane, select Create a resource.
+
+- In the search box, type “SQL 2019 on Ubuntu Pro”, and submit the search.
+
+- Select thew image by the name “SQL 2019 on Ubuntu Pro 20.04 LTS with 24x7 Support”.
+
+- Select Create on the appropriate image from the list.
+
+
+Launch the VM
+~~~~~~~~~~~~~~
+
+- Select your Subscription and Resource Group.
+
+- In Virtual machine name, enter a name for your new VM.
+
+- Then, type or select the following values:
+
+    a. Region: Select the Azure region that’s right for you.
+    b. Availability options: Choose the availability and redundancy option that’s best for your apps and data.
+    c. Change size: Select this option to pick a machine size and when done, choose Select., please note that there are some recommended sizes.
+    d. Authentication type: Select SSH public key.
+    e. SSH public key: Enter your RSA public key.
+    f. Public inbound ports: Choose Allow selected ports and pick the SSH (22) port in the Select public inbound ports list. In this tutorial, this step is necessary to connect and complete the SQL Server configuration. If you want to remotely connect to SQL Server, you will need to manually allow traffic to the default port (1433) used by Microsoft SQL Server for connections over the Internet after the virtual machine is created.
+
+- Make any changes you want to the settings in the following additional steps or keep the default settings.
+
+    a. Disks
+    b. Networking
+    c. Management
+    d. Guest config
+    e. Tags
+
+- Select Review + create.
+
+- In the Review + create pane, select Create.
+
+
+Config and connect
+------------------
+
+
+
+Connect to the VM
+~~~~~~~~~~~~~~~~~~
+
+- Find the IP address of your VM in the portal, Overview section.
+
+- Connect to the Azure VM using the ssh command. In the following command, replace the VM user name and IP address to connect to your Linux VM.
```

Comment:
Isn't the default username for ubuntu cloud images `ubuntu`?

Reply:
Do you think the example ssh command should use 'ubuntu@20.55.55.555 instead?